modules/core/src/v2/wallet.ts

```diff
@@ -484,6 +487,53 @@ export interface DownloadKeycardOptions {
   backupKeyID?: string;
 }
 
+// #region TSS interfaces
+
+export interface PrebuildTransactionWithIntentOptions {
+  reqId?: RequestTracer;
+  intentType?: string;
+  sequenceId?: string;
+  recipients?: {
+    address: string;
+    amount: string | number;
+  }[];
+  comment?: string;
+  memo?: Memo;
+  verification?: VerificationOptions;
+}
+
+export interface TSSSendManyOptions extends SendManyOptions {
+  reqId: RequestTracer;
+  intentType: string;
+}
+
+// find a better name
```

Comment:
Please add TODOS for this.

Reply:
Most comments are just placeholders so we can discuss around. This is not final at all.